File: HackerRank/Python/Easy/E0010.py
# Problem Statement: https://www.hackerrank.com/challenges/python-lists/problem

n = int(input())
l = []
for _ in range(n):
    cmd = input().split()
    if cmd[0] == 'print':
        print(l)
    else:
        getattr(l, cmd[0])(*(map(int, cmd[1:])))

File: HackerRank/C++/Easy/E0010.cpp
/*
Problem Statement: https://www.hackerrank.com/challenges/c-tutorial-stringstream/problem
*/

#include <iostream>
#include <string>
#include <sstream>

using namespace std;

int main() {
	string s, token;
	cin >> s;
	istringstream ss(s);
	while (getline(ss, token, ','))
		cout << token << endl;
	return 0;
}